src/provider/FDC3.ts

```diff
@@ -360,27 +360,27 @@ export class FDC3 {
         });
     }
 
-    private async sendContext(context: ContextBase): Promise<void> {
-        return fin.InterApplicationBus.publish('context', context);
+    private async sendContext(context: ContextBase): Promise<any> {
+        return this.apiHandler.channel.publish('context', context);
     }
 
-    private async sendIntent(intent: RaiseIntentPayload, targetApp: AppMetadata): Promise<void> {
+    private async sendIntent(intent: RaiseIntentPayload, targetApp: AppMetadata): Promise<any> {
```

Comment:
Do we want to change the signature? I feel like a change to our internal messaging system shouldn't change `void` to `any`. Could do this.apiHandler.channel.publish('intent', intent).then(() => {});

Reply:
Ditto for `sendContext`

Reply:
+1 - if the return value isn't used, should keep the return type as `void`.

Methods are `async` already, replace the `return`s with `await`s?..